src/core/HorizontalTabs/HorizontalTab.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { HorizontalTab } from './HorizontalTab';
+
+it('should render in its most basic state', () => {
+  const { container } = render(<HorizontalTab label='Tab label' />);
+  expect(container.querySelector('button.iui-tab')).toBeTruthy();
+
+  const label = container.querySelector('.iui-tab-label') as HTMLElement;
+  expect(label).toBeTruthy();
+  expect(label.firstElementChild?.textContent).toBe('Tab label');
+});
+
+it('should render with sublabel', () => {
+  const { container } = render(
+    <HorizontalTab label='Tab label' sublabel='Sub-label' />,
+  );
+  expect(container.querySelector('button.iui-tab')).toBeTruthy();
+
+  const label = container.querySelector('.iui-tab-label') as HTMLElement;
```

Comment:
maybe check the `> div` selector in here, to be specific

Reply:
Done